lib/elixir/src/elixir_interpolation.erl

```diff
@@ -78,7 +78,13 @@ unescape_tokens(Tokens) ->
   unescape_tokens(Tokens, fun unescape_map/1).
 
 unescape_tokens(Tokens, Map) ->
-  [unescape_token(Token, Map) || Token <- Tokens].
+  try [unescape_token(Token, Map) || Token <- Tokens] of
+    Unescaped ->
+      {ok, Unescaped}
```

Comment:
so, with this change to return an `ok | error` tuple, i needed to update Kernel and Macro slightly to respect this change as well.

Reply:
Yes, exactly. You should also change the macro ones to raise on error, as otherwise it would be a breaking change. So it would be something like:

```elixir
case :elixir_interpolation.unescape_tokens(...) do
  {:ok, value} -> value
  {:error, reason} -> raise ArgumentError, to_string(reason)
end
```

Reply:
that makes sense. this is a small change, i can get this updated PR in soon. 👍